drivers/py/pes/xtb.py

```diff
@@ -0,0 +1,83 @@
+"""
+Interface with tblite to provide GFN1-xTB and GFN2-xTB calculators.
+
+Example:: 
+
+    python driver.py -m xtb -u -o '{"method": "GFN2-xTB", "numbers": [8,1,1], "periodic": false}'
+"""
+
+import numpy as np
+import json
+
+from ipi.utils.units import unit_to_internal, unit_to_user
+
+try:
+    import tblite.interface as tb
+except ImportError as e:
+    raise ModuleNotFoundError(
+        "Could not find tblite for xtb driver. Please install tblite-python with mamba"
```

Comment:
This msg is not consistent with the pip installation that is suggested in the README (pip installation). What is best?